src/events.ts

```diff
@@ -84,8 +93,17 @@ export interface UserRefreshEvent extends BaseEvent {
  */
 export interface CheckUserEvent extends BaseEvent {
   type: EventType.CHECK_USER;
+  poolConfig: PoolConfig;
   /**
    * The user to check.
    */
   userId: string;
 }
+
+/**
+ * Event to run a database migration.
+ */
+export interface DBMigrationV2Event extends BaseEvent {
+  type: EventType.DB_MIGRATION_V2;
+  poolConfigs: PoolConfig[];
+}
```

Comment:
this should not be a worker event, IMO.

`refreshUser` should have the logic necessary to handle when no poolId is provided. This could probably also be handled during the migration process, as we could probably determine the previous poolId the bot was tracking, and just fill it in as a default?